src/udev/udev-builtin-net_id.c

```diff
@@ -38,6 +38,7 @@
 #include "udev-builtin.h"
 
 #define ONBOARD_INDEX_MAX (16*1024-1)
+#define ONBOARD_16BIT_INDEX_MAX (64*1024-1)
```

Comment:
gven that the 16bit one is the new one that matters, and the 14bit one is the old one, I think ONBOARD_INDEX_MAX should be 2^16-1, and then ONBOARD_14BIT_INDEX_MAX or so be defined.

Also, to make this easier to grok, maybe write out as `((1U << 14)-1U)` and `((1U << 16)-1U)`

Reply:
alternatively, name both defines with the bit valu ein them, i.e. ONBOARD_16BIT_INDEX_MAX + ONBOARD_14BIT_INDEX_MAX